packages/connect/src/index.ts

```diff
@@ -130,11 +130,7 @@ const init = async (settings: Partial<ConnectSettings> = {}) => {
     if (_core) {
         throw ERRORS.TypedError('Init_AlreadyInitialized');
     }
-    _settings = parseConnectSettings({ ..._settings, ...settings });
-    // set defaults for node
-    _settings.origin = 'http://node.trezor.io/';
```

Comment:
this was never used in node, right?

Reply:
well, it was used to set `trustedHost` lated in DataManager, but now is no longer necessary since default settings are set to `node` with `trustedHost` true